ao_experimental/quant_primitives.py

```diff
@@ -0,0 +1,43 @@
+import torch
+from torch import _dynamo
+
+def safe_int_mm(x_int8: torch.Tensor, w_int8: torch.Tensor):
```

Comment:
nit: can we be consistent with https://pytorch.org/docs/stable/generated/torch.mm.html in terms of argument names?  It's not always weight and bias.